util/build.gradle

```diff
@@ -5,6 +5,8 @@ plugins {
 dependencies {
   api 'org.bouncycastle:bcprov-jdk15on'
 
+  implementation project(':logging')
```

Comment:
This is a very unfortunate dependency to add.

Reply:
How so? ...currently `logging` is pretty much a leaf dependency (only depending on `tuweni-bytes`),  or is it more that things are being done in `utils` shouldn't be needing the new logging?

Reply:
I was living in my happy little world where `util` was a relatively small module - keep forgetting that it's a giant monolith of random stuff...

Reply:
True, `util` does quite a bit, but as I've made in the inversion now 🤷‍♂